src/eventshub/src/component/Autocomplete/Autocomplete.tsx

```diff
@@ -1,132 +1,134 @@
-import { createElement, Fragment, useEffect, useRef, useState } from 'react';
+import { createElement, Fragment, useEffect, useRef } from 'react';
 import {
   autocomplete,
   AutocompleteOptions,
   getAlgoliaResults,
 } from '@algolia/autocomplete-js';
-import { BaseItem } from '@algolia/autocomplete-core';
 import type { SearchClient } from 'algoliasearch/lite';
 import { usePagination, useSearchBox } from 'react-instantsearch-core';
+import { nanoid } from 'nanoid';
+import { useAppState } from '../../hooks';
+import { algoliaSuggestionsIndexMap } from '../../redux/slices/appSlice';
+import type { AutocompleteComponents } from '@algolia/autocomplete-js';
 import type { Root } from 'react-dom/client';
 import { createRoot } from 'react-dom/client';
+import { AlgoliaVenueItem } from '../../utilities/types';
 
 // TODO: Determine if we should build upon or remove algolia styles
-// import '@algolia/autocomplete-theme-classic';
+import '@algolia/autocomplete-theme-classic';
 
-type AutocompleteProps = Partial<AutocompleteOptions<BaseItem>> & {
+type AutocompleteProps = Partial<AutocompleteOptions<AlgoliaVenueItem>> & {
   searchClient: SearchClient;
   searchIndex: string;
-  filtersQuery?: string;
 };
 
-type SetInstantSearchUiStateOptions = {
-  query: string;
-};
+type SuggestionHitProps = {
+  hit: AlgoliaVenueItem;
+  components: AutocompleteComponents;
+}
+
+/**
+ * Suggestion hit component.
+ */
+const SuggestionHit = ({ hit, components }:SuggestionHitProps ) => {
+  return (
+    <div className="aa-ItemWrapper">
+      <div className="aa-ItemContent">
+        <div className="aa-ItemContentBody">
+          <div className="aa-ItemContentTitle">
+            <components.Highlight hit={hit} attribute="name" />
+          </div>
+          {/* <div className="aa-ItemContentDescription">
+            <strong>{hit.space_name}</strong>
+          </div> */}
```

Comment:
Might have to look into this for venues because of the `munger` `munger` `munger` problem.